xlml/apis/task.py

```diff
@@ -155,23 +171,25 @@ def provision(self) -> Tuple[DAGNode, airflow.XComArg, airflow.XComArg]:
             self.task_test_config.benchmark_id, self.tpu_name_env_var
         )
         ssh_keys = ssh.generate_ssh_keys()
-
+      
+      output_location = name_format.generate_gcs_file_location(
```

Comment:
Can you move this into the `initialize` subgroup? The other tasks are similarly generating random data that is used by the rest of the test